### DISCUSSION TITLE
Cüzdan şifre unutma

### DISCUSSION THREAD

**Discussion by abdurrahman65** [IBAX-io/go-ibax#1644] [Soru Cevap - Türkçe]
Cüzdanın şifresini unutunca daha kolay bir yöntemle şifre sıfırlama özelliği gelse iyi olur katılanlar beğenirse iyi olur

**Comment by Bulent3407** [comment]
Evet olabilir aslında

**Comment by Yusufknk** [comment]
Cüzdan ve sistem geliştirilme aşamasında

**Comment by Prosodie45** [comment]
Private keylerin hiç biriyle giriş yapılamıyor, eski de yeni de aynı hatayı veriyor. Kelimeler ve şifreler doğru. Sistemle alakalı bir sorun var değil mi?
